core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -148,6 +165,16 @@ import scala.concurrent.duration._
       log.debug("Resetting pid {}. TagProgress {}", pid, tp)
       become(idle(buffer.filterNot(_._1.persistenceId == pid), tagPidSequenceNrs + (pid -> tagPidSequenceNr)))
       sender() ! ResetPersistenceIdComplete
+
+    case ReceiveTimeout =>
+      if (buffer.isEmpty && tagPidSequenceNrs.isEmpty)
```

Comment:
So we only passivate if we have zero tagPidSequenceNrs which means there are no persistent actors active that use this tag. I'm guessing this will never be the case for Lagom but others use tags differently